tests/cypress/latest/e2e/providers_setup.spec.ts

```diff
@@ -14,11 +14,13 @@ limitations under the License.
 import '~/support/commands';
 import {qase} from 'cypress-qase-reporter/mocha';
 
+const buildType = Cypress.env('chartmuseum_repo') ? 'dev' : 'prod';
+
```

Comment:
Nit: If we add this line here:
```suggestion
const isDevBuild = buildType === 'dev';
```

we can simplify later ifs to just `if (isDevBuild) {}`